src/stylesheets/components/_alerts.scss

```diff
@@ -1,3 +1,39 @@
+$alert-slim-icon-size: units(3);
+$alert-icon-optical-factor: calc(#{units($theme-alert-icon-size)} / 6);
+$alert-icon-optical-padding: calc(
+  #{units($theme-alert-padding-x)} - #{$alert-icon-optical-factor}
+);
+
+@mixin add-alert-icon($name, $color, $bgcolor) {
+  $this-icon-object: map-merge(
+    $icon-object,
+    (
+      "name": $name,
+      "color": $color,
+      "height": $theme-alert-icon-size,
+    )
+  );
+  &:before {
+    @include add-color-icon($this-icon-object, $bgcolor);
+    // Position centered in the alert
+    bottom: 0;
+    content: "";
+    display: block;
+    // padding - optical spacing value
+    left: $alert-icon-optical-padding;
+    position: absolute;
+    height: auto;
+    top: 0;
```

Comment:
This changes the icon positioning a bit, so the icon is vertically centered on the alert.

Reply:
IE11 doesn't support `calc()` on generated content.

Maybe we should try a `left` value before this one or something like:
```
// Some left value for ie11
left: 1rem;

// Calc for modern browsers
@supports (left: $alert-icon-optical-padding) {
  left: $alert-icon-optical-padding;
}
```

![image](https://user-images.githubusercontent.com/3385219/110979594-6c134780-832a-11eb-9b7a-e2bd75c57c48.png)

Reply:
It doesn't look like the calc() is working for the content `padding-left` either

Reply:
Just removed the calc()s and used Sass math instead